node/node.go

```diff
@@ -840,14 +841,29 @@ func NtpCheckWithLocal(n *Node) error {
 		return err
 	}
 
+	ntpRetryTime := time.Duration(1)
+	var remote *time.Time
+	for i := 0; ; i++ {
+		time.Sleep(ntpRetryTime)
+		remote, err = ntpclient.GetNetworkTime(url, portNum)
+		if remote != nil {
+			break
+		}
+		// remote == nil && err != nil && i >= retry
+		if i >= retry {
+			return err
+		}
+		ntpRetryTime = ntpRetryTime * 2
```

Comment:
```suggestion
		ntpRetryTime = ntpRetryTime * 2
		time.Sleep(ntpRetryTime)
```

Reply:
@blukat29 the sleep at the head of loop is intended for immediate restart.